test-requirements.txt

```diff
@@ -3,6 +3,7 @@ pre-commit
 pytest>=6.2.4
 pytest-mock
 pytest-cov==2.10.0
+urllib3<1.27
```

Comment:
I wonder if there is a way to fix this without this pin.

cc: @ramonpetgrave64 @resilience-jychp for viz

Reply:
We'll need the pin, but on types-requests, instead.

The new types-requests version is requiring urllib>2

```
https://github.com/lyft/cartography/issues/11 59.90 Collecting urllib3>=2 (from types-requests->-r test-requirements.txt (line 8))
https://github.com/lyft/cartography/issues/11 59.93   Downloading urllib3-2.0.7-py3-none-any.whl.metadata (6.6 kB)
```

* https://github.com/typeshed-internal/stub_uploader/blob/main/data/changelogs/requests.md#23107-2023-10-01

Reply:
Thanks!